wagyu/main.rs

```diff
@@ -27,15 +30,24 @@ fn main() -> Result<(), CLIError> {
             EthereumCLI::new(),
             MoneroCLI::new(),
             ZcashCLI::new(),
+            SubCommand::with_name("update").about("Auto update to latest version"),
```

Comment:
nit: can you change the description to read `Update wagyu to the latest version`?

Reply:
Can you formalize this into an `UpdateCLI` struct, so that the command's functionality is encapsulated under the `parse` and `print` model we have defined?

This will enable future extensibility of the update command without introducing complexity outside of the Struct's scope.